cmd/kubectl-gadget/deploy.go

```diff
@@ -346,6 +96,586 @@ type parameters struct {
+							Properties: map[string]apiv1.JSONSchemaProps{
+								"apivVersion": apiv1.JSONSchemaProps{
+									Description: "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources",
+									Type:        "string",
+								},
+								"kind": apiv1.JSONSchemaProps{
+									Description: "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds",
+									Type:        "string",
+								},
+								"metadata": apiv1.JSONSchemaProps{
+									Type: "object",
+								},
+								"spec": apiv1.JSONSchemaProps{
+									Description: "TraceSpec defines the desired state of Trace",
+									Properties: map[string]apiv1.JSONSchemaProps{
+										"filter": apiv1.JSONSchemaProps{
+											Description: "Filter is to tell the gadget to filter events based on namespace, pod name, labels or container name",
+											Properties: map[string]apiv1.JSONSchemaProps{
+												"containerName": apiv1.JSONSchemaProps{
+													Description: "ContainerName selects events from containers with this name",
+													Type: "string",
+												},
+												"labels": apiv1.JSONSchemaProps{
+													Description: "Labels selects events from pods with these labels",
+													// TODO
+													// additionalProperties:
+													// type: string
+													Type: "object",
+												},
+												"namespace": apiv1.JSONSchemaProps{
+													Description: "Namespace selects events from this pod namespace",
+													Type: "string",
+												},
+												"podname": apiv1.JSONSchemaProps{
+													Description: "Podname selects events from this pod name",
+													Type: "string",
+												},
+
+											},
+											Type: "object",
+										},
+										"gadget": apiv1.JSONSchemaProps{
+											Description: "Gadget is the name of the gadget such as \"seccomp\"",
+											Type: "string",
+										},
+										"node": apiv1.JSONSchemaProps{
+											Description: "Node is the name of the node on which this trace should run",
+											Type: "string",
+										},
+										"output": apiv1.JSONSchemaProps{
+											Description: "Output allows a gadget to output the results in the specified location. * With OutputMode=Status|Stream, Output is unused * With OutputMode=File, Output specifies the file path * With OutputMode=ExternalResource, Output specifies the external   resource (such as   seccompprofiles.security-profiles-operator.x-k8s.io for the seccomp gadget)",
```

Comment:
Break this long line?

Reply:
I prefer to leave it like this as it is easier for someone to grep it.